api/src/main/java/com/clubobsidian/dynamicgui/api/function/Function.java

```diff
@@ -160,6 +160,11 @@ public Function clone() {
         }
     }
 
+    /**
+     * Checks if the function is equal to another object
+     * @param o
+     * @return true if the function is equal to another object
+     */
```

Comment:
I don't think `equals` needs a comment since it doesn't do anything outside of standard behavior.